Ginger/Ginger/WizardLib/WizardWindow.xaml.cs

```diff
@@ -265,7 +265,7 @@ private void UpdatePrevNextButton()
                 xNextButton.IsEnabled = true;
             }
 
-            if (mWizard.IsFirstPage())
+            if (mWizard.IsFirstPage() || mWizard.IsLastPage())
```

Comment:
WizardWindow is generic for all wizards, why to remove back button when user is in last wizard page?
if that is needed in Upload to shared repo wizard so you need to remove the back button only there and not for all wizards